portafly/src/i18n/locales/en/product.yml

```
@@ -0,0 +1,30 @@
+---
+# ------------------ Product > Create product
+# Page details
+
+page_title: Create product #--- PROPOSED
+page_title_description: Page title to create a product
+body_title: Create a new product #--- PROPOSED
+body_title_description: Body title of the page to create a new product
+
+# Main area
+
+name: Name
+name_description: Field label to show/input the name of the product.
+system_name: System name
+system_name_description: Field label to show/input the system name of the product.
+description: Description
+description_label: Field label to show/input the description of the product.
+#-------# save_button: available in shared.yml   #----PROPOSED
+#-------# cancel_button: available in shared.yml
+
+# Validation errors
+required_name: "<i>Name/i> is empty. You must specify a name for the product." #----PROPOSED
```

Does this need quotation marks? Is it missing a variable if it does require them?

---

Yes, it does. Quotation marks are required when using variables and format.

---

I think it doesn't, really. Neither for variables or tags: https://github.com/3scale/porta/pull/2042/files?file-filters%5B%5D=.yml#diff-36a37425608f6363a43a6ccd849d1bc9R12

---

@damianpm Has this changed? :point_up:

---

No, it doesn't need the quotation marks, but I remember @porueesq and I have talked about it, and we tought it was better to use it only when variables and format were present.
If you all agree is better without them, then OK for me.
By the way, there is a  typo:
```suggestion
required_name: "<i>Name</i> is empty. You must specify a name for the product." #----PROPOSED
```

---

Thanks @damianpm , both for the comment and the typo. I'll remove the guideline about quotation marks.

For the sake of simplicity, let's remove quotation marks with variables and formats. CC: @dfennessy